primitives/src/sequence.rs

```diff
@@ -171,6 +171,13 @@ impl Sequence {
         }
     }
 
+    /// Gets the u16 value of the last 16 bits of the sequence number.
+    /// Ideally the Block or Time lock.
+    ///
+    /// Note: Calling this on a sequence lower than [`MIN_NO_RBF`] will give you an undefined value.
+    #[inline]
+    pub const fn get_raw_value(&self) -> u16 { ((self.0 << 16) >> 16) as u16 }
+
```

Comment:
I thinks this function its not necessary at all but, for the ones who know what is doing with sequence, can be a shortcut for validation removing the necessity to cast it to a locktime